2025-03-29 16:02:06,050 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1688, in setup_module_map
    apps = get_installed_apps(_ensure_on_bench=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/caching.py", line 45, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1575, in get_installed_apps
    installed = json.loads(db.get_global("installed_apps") or "[]")
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1005, in get_global
    return self.get_default(key, user)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1009, in get_default
    d = self.get_defaults(key, parent)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1025, in get_defaults
    defaults = frappe.defaults.get_defaults_for(parent)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/defaults.py", line 230, in get_defaults_for
    defaults = frappe.cache.hget("defaults", parent)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 234, in hget
    value = pickle.loads(value)
            ^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'frappe.types.frappedict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-29 16:02:08,077 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1688, in setup_module_map
    apps = get_installed_apps(_ensure_on_bench=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/caching.py", line 45, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1575, in get_installed_apps
    installed = json.loads(db.get_global("installed_apps") or "[]")
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1005, in get_global
    return self.get_default(key, user)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1009, in get_default
    d = self.get_defaults(key, parent)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/database/database.py", line 1025, in get_defaults
    defaults = frappe.defaults.get_defaults_for(parent)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/defaults.py", line 230, in get_defaults_for
    defaults = frappe.cache.hget("defaults", parent)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 234, in hget
    value = pickle.loads(value)
            ^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'frappe.types.frappedict'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-29 16:10:59,428 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-29 16:11:07,221 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-29 16:16:25,721 ERROR frappe Error while inserting deferred Error Log record: Error Log s6ohfhrcur: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-29 16:16:25,724 ERROR frappe Error while inserting deferred Error Log record: Error Log s6ohdueaco: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-29 16:16:45,067 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-29 16:17:42,345 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-29 16:23:28,714 ERROR frappe Error while inserting deferred Error Log record: Error Log sasmp8899d: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-29 16:23:28,718 ERROR frappe Error while inserting deferred Error Log record: Error Log sasnjd1fki: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-29 16:50:29,403 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-29 16:55:05,114 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 17:45:17,860 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 17:45:20,471 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 17:51:54,049 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 17:51:56,694 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 18:06:16,809 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'helpdesk'}
2025-03-29 18:25:33,172 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-29 18:25:34,956 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-29 18:40:54,398 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'hr'}
2025-03-29 18:56:51,697 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'hr'}
2025-03-29 18:56:53,627 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'app_path': 'hr'}
2025-03-31 10:57:24,228 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 11:32:17,165 ERROR frappe Unable to load translations
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 155, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 237, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 147, in _merge_translations
    all_translations = get_translations_from_apps(lang).copy()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 173, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 185, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'
2025-03-31 11:32:17,169 ERROR frappe Unable to load translations
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 155, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 237, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 147, in _merge_translations
    all_translations = get_translations_from_apps(lang).copy()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 173, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 185, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'
2025-03-31 11:32:18,473 ERROR frappe Unable to load translations
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 155, in get_all_translations
    return frappe.cache.hget(MERGED_TRANSLATION_KEY, lang, generator=_merge_translations)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 237, in hget
    value = generator()
            ^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 147, in _merge_translations
    all_translations = get_translations_from_apps(lang).copy()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 173, in get_translations_from_apps
    translations.update(get_translations_from_csv(lang, app) or {})
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/translate.py", line 185, in get_translations_from_csv
    os.path.join(frappe.get_app_path(app, "translations"), lang + ".csv"), lang, app
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'
2025-03-31 11:34:47,032 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hrms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 12:39:54,171 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hrms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 12:42:46,989 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 13:12:45,027 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 99, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 172, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 313, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1692, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1540, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1505, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1535, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1476, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1333, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 66, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 95, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 18:22:47,163 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 100, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 173, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 258, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1652, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1500, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1465, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1495, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1436, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hrms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1299, in get_doc
    return frappe.model.document.get_doc(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 67, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 96, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 18:22:49,775 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 100, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 173, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 258, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1652, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1500, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1465, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1495, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1436, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hrms'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1299, in get_doc
    return frappe.model.document.get_doc(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 67, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 96, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 18:26:40,755 ERROR frappe Could not take error snapshot: 'NoneType' object has no attribute 'get'
Site: bizproo.com
Form Dict: {}
Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 100, in application
    init_request(request)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/app.py", line 173, in init_request
    frappe.init(site=site, sites_path=_sites_path, force=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 258, in init
    setup_module_map(include_all_apps=not (frappe.request or frappe.job or frappe.flags.in_migrate))
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1652, in setup_module_map
    for module in get_module_list(app):
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1500, in get_module_list
    return get_file_items(get_app_path(app_name, "modules.txt"))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1465, in get_app_path
    return get_pymodule_path(app_name, *joins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1495, in get_pymodule_path
    return abspath(join(dirname(get_module(scrub(modulename)).__file__ or ""), *joins))
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1436, in get_module
    return importlib.import_module(modulename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'helpdesk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 86, in log_error_snapshot
    log_error(title=str(exception), defer_insert=True)
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/utils/error.py", line 61, in log_error
    error_log = frappe.get_doc(
                ^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/__init__.py", line 1299, in get_doc
    return frappe.model.document.get_doc(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/document.py", line 83, in get_doc
    controller = get_controller(doctype)
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 67, in get_controller
    return import_controller(doctype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/model/base_document.py", line 96, in import_controller
    module = load_doctype_module(doctype, module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 248, in load_doctype_module
    app = get_module_app(module)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mbr-python/Desktop/bizproo/frappe-bench/apps/frappe/frappe/modules/utils.py", line 271, in get_module_app
    app = frappe.local.module_app.get(scrub(module))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
2025-03-31 18:49:52,360 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-31 18:50:06,190 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
2025-03-31 19:01:00,509 ERROR frappe Error while inserting deferred Error Log record: Error Log umddqi8hds: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-31 19:01:00,513 ERROR frappe Error while inserting deferred Error Log record: Error Log umdiqler01: 'Title' (Module import failed for Workspace Settings, the DocType you're trying to open might be deleted.
Error: No module named 'frappe.core.doctype.workspace_settings') will get truncated, as max characters allowed is 140
Site: bizproo.com
Form Dict: {}
2025-03-31 19:01:29,894 ERROR frappe New Exception collected in error log
Site: bizproo.com
Form Dict: {'doctype': 'Workspace Settings', 'cmd': 'frappe.client.get'}
